pkg/pow/solutionHandler.go

```diff
@@ -63,9 +62,11 @@ func (s *SolutionHandler) Submit(ctx context.Context, result *Result) (*types.Tr
 		val := m[valKey]
 		var value *big.Int
 		if len(val) == 0 {
-			cfg := config.GetConfig()
-			indexPath := filepath.Join(cfg.ConfigFolder, "manualData.json")
-			jsonFile, err := os.Open(indexPath)
+			cfg, err := config.ParseConfig("")
+			if err != nil {
+				return nil, errors.Wrapf(err, "parsing config")
+			}
+			jsonFile, err := os.Open(cfg.ManualDataFile)
```

Comment:
No we want to pass the config as a function argument, the parsing should happen only in a single place somewhere in the main.go or cmds.go

So do a global search and there should be no other places where  ParseConfig is used.

This should become 
```suggestion
			jsonFile, err := os.Open(s.cfg.ManualDataFile)
```

Reply:
yup sometimes i miss em! :D